### DISCUSSION TITLE
Help needed to send Peppol documents to locally hosted AS4 server

### DISCUSSION THREAD

**Discussion by sivadasrajan** [phax/phase4#241] [Q&A] [answered]
Hi All,
I was following this https://github.com/phax/phase4/wiki/Testing#testing-without-an-official-peppol-certificate .
And i did until the step 6, setting up instances on localhost:8080 and localhost:8081.
But I' on stuck on step 7

`7. Try sending from first phase4 instance to second phase4 instance`

It connects to https://www.helger.com/phase4/as4 just fine for sending sample invoice. But i want to send to http://localhost:8080/as4 for testing.
I am not sure what to do. How can i specify which url to hit. Do i need to configure a local smp?

**Comment by phax** [comment, marked as answer]
Hello @sivadasrajan , step 6 contained a typo. It should be 'ks2' and not ks1' - I corrected the error in the page already.

I will add some text on how to configure a predefined Url and Certificate into the Peppol Client Sender later. See https://github.com/phax/phase4/blob/master/phase4-peppol-client/src/main/java/com/helger/phase4/peppol/Phase4PeppolSender.java#L707 on the method you need to invoke to pass in the data.

**Reply by sivadasrajan** [reply]
Thanks @phax. I really appreciate the help and will try this soon
